conf/config_test.go

```diff
@@ -105,31 +105,72 @@ func TestUnsetEnv(t *testing.T) {
 	}
 }
 
-func Test_setup(t *testing.T) {
-	type args struct {
-		dir string
-	}
+func TestLoadConfigs(t *testing.T) {
```

Comment:
rename and adding a test case to make sure that when we load more than one config they're merged